gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -204,9 +204,9 @@
               <div class="row">
                 <div class="col-md-6">
                   <label>
-                    <input type="checkbox" ng-model="currentFolder.fsWatcherEnabled" ng-change="setFSWatcherIntervalDefault()" tooltip data-original-title="{{'Use notifications from the filesystem to detect changed items.' | translate }}">&nbsp;<span translate>Watch for Changes</span>
+                    <input type="checkbox" ng-model="currentFolder.fsWatcherEnabled" ng-change="setFSWatcherIntervalDefault()">&nbsp;<span translate>Watch for Changes</span>
                   </label>
-                  <p translate class="help-block">Watching for changes discovers most changes without periodic scanning.</p>
+                  <p class="help-block"><span translate>Use notifications from the filesystem to detect changed items.</span> <span translate>Watching for changes discovers most changes without periodic scanning.</p>
```

Comment:
How about wrapping each `<span>` into an own line?  The HTML is already pretty hard to read and unnecessarily long line don't help.

And it would have increased the odds of detecting your missing `</span>` closing tag ;-)